js/ui/list/ui.list.base.js

```diff
@@ -41,6 +42,10 @@ var ListBase = CollectionWidget.inherit({
 
     _activeStateUnit: LIST_ITEM_SELECTOR,
 
+    _getFeedbackSelector: function() {
```

Comment:
I don't think that extracting _getFeedbackSelector  is a good idea - feedback and active state are the same states. Modify _activeStateUnit instead.